Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -871,7 +895,99 @@ private void addExtensionTypeSettings(ArrayList<SettingsItem> sl, int wiimoteNum
   {
     switch (extentionType)
     {
-      case 1: // Nunchuk
+      case SettingsActivityPresenter.WIIMOTE_EXT_NONE:
+        final int wiimoteNumberAdjusted;
+
+        if (mSettings.isGameSpecific())
+        {
+          wiimoteNumberAdjusted = wiimoteNumber - 4;
+          mSettings.loadWiimoteProfile(mGameID, wiimoteNumberAdjusted);
+        }
+        else
+        {
+          wiimoteNumberAdjusted = wiimoteNumber - 3;
+        }
+
+        AbstractBooleanSetting wiimoteOrientation = new AbstractBooleanSetting()
+        {
+          Boolean valueSelected;
+
+          @Override
+          public boolean getBoolean(Settings settings)
+          {
+            if (valueSelected != null)
+            {
+              return valueSelected;
+            }
+            else
+            {
+              return getWiimoteSection(wiimoteNumberAdjusted, true)
+                      .getBoolean(SettingsFile.KEY_WIIMOTE_ORIENTATION, false);
+            }
+          }
+
+          @Override
+          public void setBoolean(Settings settings, boolean newValue)
+          {
+            // Set this setting now since multiple settings that change controllers may be modified.
+            SharedPreferences preferences = PreferenceManager
+                    .getDefaultSharedPreferences(mView.getActivity().getApplicationContext());
+            final SharedPreferences.Editor editor = preferences.edit();
+
+            editor.putInt("wiiController", newValue ? InputOverlay.OVERLAY_WIIMOTE_HORIZONTAL :
+                    InputOverlay.OVERLAY_WIIMOTE_VERTICAL);
+            editor.commit();
+
+            getWiimoteSection(wiimoteNumberAdjusted, false).setBoolean(settings.isGameSpecific() ?
+                    SettingsFile.KEY_WIIMOTE_ORIENTATION + wiimoteNumberAdjusted :
+                    SettingsFile.KEY_WIIMOTE_ORIENTATION, newValue);
+
+            valueSelected = newValue;
+            mView.onSettingChanged();
```

Comment:
There's no need to call `onSettingChanged` here and in `delete`. The calling code will call it.